app/javascript/components/student/IterationsList.tsx

```diff
@@ -109,9 +109,7 @@ export const IterationsList = ({
     }
 
     if (isOpen.length === 0) {
-      setIsOpen(
-        resolvedData.iterations.map((iteration, i) => (i === 0 ? true : false))
-      )
+      setIsOpen(resolvedData.iterations.map((_, i) => (i === 0 ? true : false)))
```

Comment:
This should be the same:

```suggestion
      setIsOpen(resolvedData.iterations.map((_, i) => i === 0)))
```

Reply:
I probably auto-formatted this older bit. thanks for catching it!